en/docs/administration/configure-a-custom-domain-for-apis.md

```diff
@@ -0,0 +1,49 @@
+# Configure a Custom Domain for APIs
+
+A custom domain for your site is essential for effective branding, discoverability, and credibility. Choreo allows you to configure a custom domain for the APIs you deploy in it.
+
+To understand how to configure a custom domain for your APIs, follow the steps in the following sections:
+
+## Prerequisite
+
+To configure a custom domain for your APIs, you must first register a custom domain with a domain registrar service as follows:
+
+
+1. Sign in to the Choreo Console at <a>https://console.choreo.dev/</a> using a Google/ GitHub/ Microsoft account.
```

Comment:
Let's make the console a link with mkdocs syntax

Reply:
Done